tests_trung/utils/constants.py

```diff
@@ -0,0 +1,11 @@
+#site constants
+DATE_FORMAT = "%Y-%b-%d"
+DA_URL = "http://localhost:55000/TADashboard/login.jsp"
```

Comment:
Should have a way to update this variable in real-time. When I need to run all of tests on other URL. What should I do?